ddp-workspace/projects/ddp-dsm-ui/src/app/participant-list/participant-list.component.html

```diff
@@ -734,6 +734,29 @@ <h3 class="Color--primary Line--Break">{{additionalMessage}}</h3>
                       </li>
                     </ul>
                   </div>
+                  <div *ngIf="alias === 'c'">
+                    <ul class="NO-PADDING-START">
+                      <li class="LIST-SEPARATOR" *ngFor="let tag of pt.data.dsm.cohortTag">
+                        <p class="BIT-OF-MARGIN">
+                          <ng-container *ngIf="col.participantColumn.object !== 'testResult'">
+                            <ng-container
+                              *ngIf="col.type !== 'DATE' && col.type !== 'BOOLEAN' && col.type !== 'OPTIONS' && col.type !== 'CHECKBOX'">{{tag[col.participantColumn.name]}}
```

Comment:
**_#minor_**
The condition 
`col.type !== 'DATE' && col.type !== 'BOOLEAN' && col.type !== 'OPTIONS' && col.type !== 'CHECKBOX'`
can be simplified like 
`!['DATE', 'BOOLEAN', 'OPTIONS', 'CHECKBOX'].includes(col.type)`